spec/integration/publish_intent_crud_spec.rb

```diff
@@ -23,14 +23,11 @@
         expect(intent.publish_time).to match_datetime(publish_time)
       end
 
-      it "responds with a created status, and the intent as json" do
+      it "responds with a created status and an empty JSON document" do
         put_json "/publish-intent/vat-rates", data
 
         expect(response.status).to eq(201)
-
-        data = JSON.parse(response.body)
-        expect(data['base_path']).to eq('/vat-rates')
-        expect(data['publish_time']).to match_datetime(publish_time)
+        expect(response.body).to eq('{}')
```

Comment:
given [Daniel's comment on the API guidelines RFC](https://gov-uk.atlassian.net/wiki/display/PUB/RFC+6%3A+API+guidelines?focusedCommentId=23855338#comment-23855338), I'll go ahead and merge this